scootapi/api.go

```diff
@@ -1,3 +1,5 @@
 package scootapi
 
-// We should use go generate to run thrift --gen go:package_prefix=github.com/scootdev/scoot/scootapi/gen-go/ scoot.thrift
+// We should use go generate to run:
+//   thrift --gen go:package_prefix=github.com/scootdev/scoot/scootapi/gen-go/ scoot.thrift
```

Comment:
I'm sorry, I think I may have forgotten to update this. Thrift has an option so I think you can just do:

//   thrift --gen go:package_prefix=github.com/scootdev/scoot/scootapi/gen-go/,thrift_import=github.com/apache/thrift/lib/go/thrift scoot.thrift